source/models/evaluate_model.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Module for evaluating binary classification models of tweets authenticity.
+
+June, 2019
+@author: Joshua Rubin
+"""
+
+import os
+import json
+import numpy as np
+import pandas as pd
+
+from sklearn.metrics import confusion_matrix
+from averaged_cosine_sim_model import AvgCosSimModel
+
+CONFIG_PATH = "../../config.json" 
+
+def load_tweets_from_directory(directory_path, trin_test_split_frac = 0.5):
+    """ Pull in tweet data by user from <directory_path>, shuffle, split.
+        Returns tuple of dataframes, trining and test.
+    """
+    frames = []
+    for file in os.listdir(directory_path):
+        if file[0] == '@':
+            print(file)
+            newFrame = pd.read_json(os.path.join(directory_path,file))
+            newFrame.columns = ['tweet','date','embedding']
+            newFrame['name'] = file.split('.')[0]
+            frames.append(newFrame)
+        
+    allData = pd.concat(frames)
+
+    return np.split(allData.sample(frac=1),
```

Comment:
```
from sklearn.model_selection import train_test_split

train, test = train_test_split(df, test_size=0.2)
```

Reply:
Don't need to write everything yourself. It's also easier for other people because they don't have to wonder if your homemade split works correctly